packages/reputation-miner/ReputationMinerClient.js

```diff
@@ -125,8 +129,25 @@ class ReputationMinerClient {
         }
       });
 
+      // Query all reputation for a single user in a colony
+      this._app.get("/:rootHash/:colonyAddress/all/:userAddress", cache('1 hour'), async (req, res) => {
```

Comment:
I'd have a mild preference for `/:rootHash/:colonyAddress/:userAddress/all` if you're amenable to that